display/d.labels/do_labels.c

```diff
@@ -84,7 +84,7 @@ int do_labels(FILE *infile, int do_rotation)
         else if (!strncmp(text, "yof", 3))
             sscanf(text, "%*s %d", &yoffset);
         else if (!strncmp(text, "col", 3)) {
-            sscanf(text, "%*s %s", buff);
+            sscanf(text, "%*s %127s", buff);
```

Comment:
You could use number (SCHAR_MAX, UCHAR_MAX) from `limits.h`  instead of using "magic number" for 127, 255, etc

Reply:
so you are suggesting to change 127 to SCHAR_MAX and 255 to UCHAR_MAX.
And in that case should i leave 49 and 49 for word1 and word2 as it is?

Reply:
I agree it would be a good idea to avoid "magic numbers", but we may just as well define them ourself (e.g. `#define BUFFSIZE 128`) – no need for limits.h constants which are not really related. To make full use of the defined constants, we must also dynamically create the formats given to `sscanf()` (e.g. `snprintf(buff_fmt, sizeof(buff_fmt), "%%*s %%%ds", BUFFSIZE - 1);`).
We can use `GNAME_MAX` (defined in gis.h) instead of the value `256`.

Reply:
did changes but CI check is failing

Reply:
> did changes but CI check is failing

You can always look at the "Details" of each CI runner to see what caused the failure in the build log.

Reply:
Ok